scripts/index.js

```diff
@@ -0,0 +1,18 @@
+const menuBars = document.querySelector('.menu-bars');
+const menuX = document.querySelector('.menu-x');
```

Comment:
- [ ] [OPTIONAL] The mobile menu is appearing as expected and the links take you to the correct part of the page, but the mobile menu is not closing when clicking the 'X'. I truly apologize but there has been a mistake made in the previous review since this is one of the requirements:
![image](https://user-images.githubusercontent.com/81522435/136110985-466801b1-2267-425a-86c4-f66f1a7846d7.png)
This issue will be marked as `[OPTIONAL]` but I strongly recommend fixing this in order to avoid problems in future milestones for this project. Once again, I apologize for the inconvenience :raised_hands: